main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestCreatePhoneBook(t *testing.T) {
+	for _, tc := range []struct {
+		names   []string
+		numbers []string
+		want    map[string]string
+	}{
+		{[]string{"A1", "A2", "A3"}, []string{"+371222", "+371111", "+371333"}, map[string]string{"A1": "+371222", "A2": "+371111", "A3": "+371333"}},
```

Comment:
* It would be useful to have test descriptions here (e.g. "only unique names", "names with duplicate", etc.)
* Some tests are missing: empty slices, multiple duplicate names.